TTS/bin/extract_tts_spectrograms.py

```diff
@@ -0,0 +1,289 @@
+#!/usr/bin/env python3
```

Comment:
we need to implement a test for this script to make sure that the computed spectrograms are right (I don't know how) but at least the script works as we update the rest of the project.

Reply:
Maybe we can compare the features computed by this script with a simpler script loading the model and computing the output spectrogram for just a simple wav file using the same `config.json`.

Reply:
Please check if what I did is good enough